client/MentorPanel/src/redux/actions.js

```diff
@@ -1,4 +1,4 @@
-import { queryMentor } from '../api/api'
+import { topicsUrl, questionsUrl, queryMentor } from '../api/api'
```

Comment:
can we migrate to absolute imports for own packages? Not sure what's involved but on quick search, I found this: https://www.gatsbyjs.org/packages/gatsby-plugin-root-import/